src/main/java/higtools/modules/main/AutoCenter.java

```diff
@@ -0,0 +1,72 @@
+package higtools.modules.main;
+
+import higtools.HIGTools;
+import meteordevelopment.meteorclient.events.world.TickEvent;
+import meteordevelopment.meteorclient.settings.DoubleSetting;
+import meteordevelopment.meteorclient.settings.Setting;
+import meteordevelopment.meteorclient.settings.SettingGroup;
+import meteordevelopment.meteorclient.systems.modules.Module;
+import meteordevelopment.orbit.EventHandler;
+
+import static higtools.utils.HIGUtils.getHighway;
+
+public class AutoCenter extends Module {
+    // x+ x- z+ z- x+z+ x-z+ x+z- x-z-
```

Comment:
Wat

Reply:
That comment was there in the KT sourece too, so I just left it.
Can be removed if you want. Guess it was a reminder for all the 8 axis directions.

Reply:
Perfect thanks